src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -328,23 +342,22 @@ const HoloFuelDnaInterface = {
       const nonActionableTransactions = transactions.map(presentTransaction).filter(tx => !(tx instanceof Error))
       const uniqueNonActionableTransactions = _.uniqBy(nonActionableTransactions, 'id')
 
-      const myProfile = await HoloFuelDnaInterface.user.get()
+      // const myProfile = await HoloFuelDnaInterface.user.get()
 
       const nonActionableTransactionsWithCancelByKey = uniqueNonActionableTransactions
         .filter(tx => tx.status !== 'pending')
-        .map(tx => tx.status === STATUS.canceled ? { ...tx, canceledBy: myProfile } : { ...tx, canceledBy: null })
-
-      const presentedNonActionableTransactions = await getTxWithCounterparties(nonActionableTransactionsWithCancelByKey)
+        // nb: cancelled txs have been removed for now; keep this commented out as long as they continue being so
+        // .map(tx => tx.status === STATUS.canceled ? { ...tx, canceledBy: myProfile } : { ...tx, canceledBy: null })
```

Comment:
We can remove this because we will not be implementing cancel for this version of the DNA anyway